WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -95,6 +95,16 @@ public struct WordPressAuthenticatorConfiguration {
     /// Default value is disabled
     let continueWithSiteAddressFirst: Bool
 
+    /// If enabled shows a "Sign in with site credentials" button in `GetStartedViewController` when landing in the screen after entering site address
+    ///  Used in WooCommerce iOS to enable sign in using site credentials feature.
+    ///  Disabled by default
+    let enableSiteCredentialsLoginInGetStartedScreen: Bool
```

Comment:
Nit: We might not need to mention where the button is, but instead what the button is for:
```suggestion
    let enableSiteCredentialsLoginForSelfHostedSites: Bool
```

Reply:
Done in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/649/commits/5e7885880c8d902a807511de2b418fdb8d4972e4